lib/extras/dec/color_description.cc

```diff
@@ -210,29 +224,15 @@ Status ParseTransferFunction(Tokenizer* tokenizer, JxlColorEncoding* c) {
 Status ParseDescription(const std::string& description, JxlColorEncoding* c) {
   *c = {};
   if (description == "sRGB") {
-    c->color_space = JXL_COLOR_SPACE_RGB;
-    c->white_point = JXL_WHITE_POINT_D65;
-    c->primaries = JXL_PRIMARIES_SRGB;
-    c->transfer_function = JXL_TRANSFER_FUNCTION_SRGB;
-    c->rendering_intent = JXL_RENDERING_INTENT_PERCEPTUAL;
+    return ParseDescription("RGB_D65_SRG_Per_SRG", c);
   } else if (description == "DisplayP3") {
-    c->color_space = JXL_COLOR_SPACE_RGB;
-    c->white_point = JXL_WHITE_POINT_D65;
-    c->primaries = JXL_PRIMARIES_P3;
-    c->transfer_function = JXL_TRANSFER_FUNCTION_SRGB;
-    c->rendering_intent = JXL_RENDERING_INTENT_PERCEPTUAL;
+    return ParseDescription("RGB_D65_DCI_Per_SRG", c);
+  } else if (description == "Adobe98") {
```

Comment:
Do you remember why we chose “perceptual” for these two and “relative” for the others? Maybe “relative” for all would make sense?

Reply:
No, I don't remember. Relative for all is fine with me. Updating.